Makefile

```diff
@@ -247,9 +247,9 @@ ifneq ($(HAS_WEBAPP),)
 endif
 
 .PHONY: test-e2e
-test-e2e:
+test-e2e: dist
 	@echo Running e2e tests
-	MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
+	PLUGIN_BUNDLE=$(shell pwd)/dist/$(BUNDLE_NAME) MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
```

Comment:
Why is `MM_SERVER_PATH` is passed to the command again? Is the env variable not enough?